### DISCUSSION TITLE
Removing a star on a listed repo removes it from the list

### DISCUSSION THREAD

**Discussion by seinecle** [community/community#9315] [Other Feature Feedback, Questions, & Ideas]
Sorry I am on mobile with a bad virtual keyboard so I'll keep it short:

Removing a star on a listed repo removes it from the list. I noticed I had starred some of my own repos. Unstarring these repos, when I was on the list page (can't remember which, was on mobile), removed this repo from the list it was currently in. Not the intended outcome, right?

**Comment by brianlovin** [comment]
This is the correct behavior, but your feedback is helpful for us to know if things aren't clear enough. Right now adding a repository to a list is linked to the starring action. If you unstar a repo, it will also remove it from all lists.

**Reply by seinecle** [reply]
Thank you! 
I use lists to organize my own repos (btw, the page to these lists of my own repos is super convenient as an alternative to my github profile, to present my work). 
But it feels weird to star its own repos (as if liking its own posts on social media 😊). 

I guess starring and lists should be decoupled in a v2 or v3...

**Reply by brianlovin** [reply]
We'll consider this down the road! Also: we are adding a dialog that will make it more clear that unstarring a repository will also remove it any lists, hopefully that will help in the meantime 😊

**Comment by jessicatorreslima** [comment]
I feel the same as @seinecle. I find it very convenient using lists to organize my repositories, but I want to do it without adding stars to it.

**Reply by zombiedogz** [reply]
I agreed with others. Organizing the repo's is critical. It should not be starred. That seems silly to star a list. Having tags or directories, list, labels whatever would be very helpful. Seems like such a minor thing...I would use stars for thinks currently working on. My two cents for what that is worth lol.

**Comment by chingc** [comment]
I would love to see a Star filter that hides repos that are in a list.

When organizing my Stars I would get to the point where it was difficult to tell whether or not something was already in a list.
